teamcity: failed tests on master: acceptance/TestDockerCLI

The following tests appear to have failed:

[#26440](https://teamcity.cockroachdb.com/viewLog.html?buildId=26440):

```
--- FAIL: acceptance/TestDockerCLI (69.729s)
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	replication_test.go:46: replication test is a no-op for empty cluster
	cli_test.go:68: non-zero exit code: 1

```
Please assign, take a look and update the issue accordingly.

--- a/cli/interactive_tests/test_reconnect.tcl
+++ b/cli/interactive_tests/test_reconnect.tcl
@@ -20,7 +20,11 @@ eexpect root@
 
 send "select 1;\r"
 eexpect "opening new connection"
-eexpect "connection refused"
+expect {
+    "connection refused" {}
+    "connection reset by peer" {}
+    timeout {exit 1}
+}
 eexpect root@
 
 # Check that the client automatically reconnects when the server goes up again.

